re-docs/PROTOCOLS.md

```diff
@@ -191,6 +191,7 @@ Here is what it's currently known to contain:
 | Fan 6 substructure                 | 0xBE                     |
 | Fan 7 substructure                 | 0xCB                     |
 | Fan 8 substructure                 | 0xD8                     |
+| Flow sensor                        | 0x3D                     |
```

Comment:
`0x7B`?

Reply:
That's what I get for addressing this PR while being sleep deprived :smile: but I wanted to leverage your quick feedback!

I'll amend this commit.

Reply:
> That's what I get for addressing this PR while being sleep deprived

That makes two of us right now 😆 Don't worry, FOSS is a big part of my life and I'm not going anywhere.

Please update the kernel docs as well (note that Octo has flow sensor pulses and add an entry for fan9_pulses to the table) and we should be good to merge, I think.

Reply:
Updated kernel docs too now! (Hope I understood what you meant since the sysfs documentation is quite minimalistic)